addons/main/functions/fnc_addChestpack.sqf

```diff
@@ -17,9 +17,9 @@
  *
  * Public: No
  */
-params ["_unit", "_chestpackClass", ["_backpackLoadout", []], ["_backpackVariables", []], ["_backpackLoad", nil]];
+params ["_unit", "_chestpackClass", ["_backpackLoadout", []], ["_backpackVariables", []], ["_backpackLoad", -1]];
 
-if (_backpackLoad isEqualTo nil) then {
+if (_backpackLoad isEqualTo -1) then {
```

Comment:
A `< 0` would make it much more foolproof while indeed using the `-1` as the `params` undefined substitution.

Also, at least when I tried, `iET` was consistently slower for a number than `==`.

```suggestion
if (_backpackLoad < 0) then {
```

Reply:
> Also, at least when I tried, `iET` was consistently slower for a number than `==`.

That should not be the case. `isEqualTo` is a strict, binary comparison which should be extremely efficient and also makes it significantly stricter than `==`.

Reply:
> > Also, at least when I tried, `iET` was consistently slower for a number than `==`.
> 
> That should not be the case. `isEqualTo` is a strict, binary comparison which should be extremely efficient and also makes it significantly stricter than `==`.

totally agree

Reply:
Precisely my thinking, that's why I've been favoring `iET`/`iNET`, but I figured I'd test it. My machine has a consistently inconsistent performance (partially due to an incorrectly debloated Win10), especially with 'Arma 3', but somehow that's the result trying it about 20 times with both overall.